### ISSUE TITLE
Details on the Party Itself

### ISSUE THREAD

**Issue by nitrodragon** [nitrodragon/GameStats#6]
I'm thinking of incorporating the same system I saw in _Dragon Quest III,_ where you choose the **class, name, and gender** of the people you want in your party. From there, you can pick them up as mercenaries from the tavern that you look for them at, and they'll join your party. 

Here're some extra concerns I've got: 

- I know I want a party of **four**, preferably **two male, two female**, but it'd be a cool thing to _let the players choose the characters in the story_.
- Regarding this, I kind of want to have a force female character in there, so we can get that whole _romance_ thing happening.
- Furthermore, this means you'll only have two villagers in the game (you and your to-be ladyfriend), as we'll probably have you recruit team members **straight from Class 1.** 
- Further-furthermore, I want to incorporate **death**. Of course, a member who is alive throughout the whole game **will be better than one who isn't** _(thank you, EVs),_ but it gives the player a better experience in the game; they can experiment with multiple classes and abilities.
- Further-further-furthermore, I'm on the fence about whether or not to end the game if your love interest dies. It'd be cool to have **three slots for character creation**, but a game without some form of romance is a little, well, dry. I'll add in a touching scene if we decide she can die and she dies, and, of course, make it worth your while if you keep her around throughout the game.
- Even further-further-furthermore, I'll keep constantly updating this thread, so be sure to keep an eye on it.

**Comment by SirNectarine** [comment]
If we're going with player freedom when it comes to the party, why force a love interest? Why not allow the player to try and recruit any NPC they come across to their party? Why not allow the player to fall in love and attempt to have a romance with almost any NPC they come across (regardless of gender)? 
Of course, you can still have a default party and storyline, but if there's going to be player freedom, then we ought to go the full way, leading to an insanely in-depth and interactive RPG.
(Of course, that'd be extremely difficult to code and do right, so we'll see.)

**Comment by nitrodragon** [comment]
@SirNectarine Exactly, and that's why I kinda want to force the love interest. Makes it easier to code lol

**Comment by EthanBar** [comment]
@nitrodragon I like the laziness being planned here
